meta-resin-common/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -1,3 +1,10 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
+
+SRC_URI_append = "${@bb.utils.contains('DISTRO_FEATURES', 'debug-image', '', ' file://remove_systemd-getty-generator.patch', d)}"
+
+# remove systemd-serialgetty from a non-debug build
+RRECOMMENDS_${PN}_remove = "${@bb.utils.contains('DISTRO_FEATURES', 'debug-image', '', 'systemd-serialgetty', d)}"
```

Comment:
Actually, we can remove just the systemd-serialgetty package and we have the issue solved. So the template needs to be removed in any case, we cannot keep it